_sass/base.scss

```diff
@@ -157,6 +157,13 @@ em {
   }
 }
 
+.anchor{
```

Comment:
Please move this `Get-involved` section of the code

Reply:
Instead of moving to `Get-involved` section,  similar to https://github.com/hotosm/hotosm-website/blob/gh-pages/_sass/base.scss#L4794-L4796 - create a new one for `Volunteer` and add contents there.